plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -228,7 +228,37 @@ SecRule REQUEST_URI "@contains /apps/text/session/sync" \
     nolog,\
     ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:autosaveContent"
-        
+
+# Text app content can be anything
+SecRule REQUEST_FILENAME "@endsWith /apps/text/public/session/sync"
+   "id:9508127,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.autosaveContent"
+
+# Text app content can be anything
+SecRule REQUEST_FILENAME "@endsWith /apps/text/public/session/push"
```

Comment:
Missing ` \` at the end of line.

Reply:
the rule will not work if I add ``/`` at the end of the ``REQUEST_FILENAME``, that's how the request is made.